src/test/store/markers.test.js

```diff
@@ -121,6 +121,11 @@ describe('selectors/getMarkerChartTimingAndBuckets', function() {
       ]);
     });
   });
+
+  it('should not put network markers in the marker chart', function() {
+    const markerTiming = getMarkerChartTimingAndBuckets(getNetworkMarkers());
+    expect(markerTiming).toEqual([]);
+  });
```

Comment:
I added a new explicit test, but I realized when testing on the main branch that this was also tested below in `filters for getMarkerChartMarkerIndexes`. But because we had a "fake" schema the test wasn't testing this case obviously.

Reply:
I would ensure this works for profiles that were previously upgraded, and have duplicate schema definitions. One in the profiler, and one in the front-end provided schema. The front-end provided schema should override here.

Reply:
I added a test for this.